<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomSort</class>
 <widget class="QDialog" name="CustomSort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Sort Order</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/icon32.ico</normaloff>:/icon32.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0" colspan="2">
    <widget class="QComboBox" name="field_5"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QComboBox" name="field_4"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QComboBox" name="field_2"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QComboBox" name="field_3"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="sort_label">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;New Custom Sort Order&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="sort_name"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QComboBox" name="field_1"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter name of new custom sort:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Select order:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QComboBox" name="field_0">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QComboBox" name="field_6"/>
   </item>
   <item row="13" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>72</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QComboBox" name="field_7"/>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QComboBox" name="field_8"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
